src/app/pages/signup/signup.component.ts

```diff
@@ -52,7 +52,7 @@ export class SignupComponent {
 		if (this.previewImage) {
 			submittedUser.profilePhoto = this.previewImage;
 		}
-		this.userService.createOrUpdate(submittedUser, this.teamId)
```

Comment:
Looks great, i'm just worried about this guy here. Are you sure this is not needed? This teamId is for creating an user with team

Reply:
I took off just because there's no method in service which receives a teamId...

Reply:
holy sh#$ thats right... I did it wrong